gui/wxpython/gui_core/gselect.py

```diff
@@ -2101,10 +2102,27 @@ def GetDsn(self):
         if self._sourceType == "db":
             if self.dbWidgets["format"].GetStringSelection() in (
                 "PostgreSQL",
+                "PostgreSQL/PostGIS",
                 "PostGIS Raster driver",
             ):
-
-                dsn = "PG:dbname=%s" % self.dbWidgets["choice"].GetStringSelection()
+                ret = RunCommand("db.login", read=True, quiet=True, flags="p")
+                if not ret:
+                    GError(
+                        parent=self,
+                        message=_(
+                            "PostgreSQL/PostGIS login was not setted."
+                            " Set it via <db.login> module, please."
+                        ),
+                    )
+                    return
+                db_login = ret.split("|")
+                return (
```

Comment:
`v.external -t input="PG:dbname=<your_db_name>"` (which is used in this code) https://github.com/OSGeo/grass/blob/78e62f023a67d892157c23d26f729efb386afe0a/gui/wxpython/gui_core/gselect.py#L2232 require only PostgreSQL connection `dbname=` parameter `PG:dbname=<your_db_name>` to wok, because has ability to read all PostgreSQL connection from `db.login` module `~/.grass8/dblogin` file via [`db_get_login2()`](https://grass.osgeo.org/programming8/login_8c_source.html#l00379) func.

https://github.com/OSGeo/grass/blob/78e62f023a67d892157c23d26f729efb386afe0a/vector/v.external/dsn.c#L33

`v.import` module which is used for import PostGIS table, require full PostgreSQL connection string `"PG:dbname=<your_dbname> user=<your_db_user> password=<your_db_password> host=<your_db_host> port=<your_db_port>"`,
because has not ability to read PostgreSQL connection from `db.login` module `~/.grass8/dblogin` file and that's why it doesn't work `v.import input="PG:dbname=<your_db_name>" layer=<your_postgis_table_name> output=<your_outpur_vector_map>`